.github/workflows/pr-commands.yaml

```diff
@@ -0,0 +1,173 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+# Modifications:
+# - Allow more roles to trigger each PR command
+# - Document builds README.md from README.Rmd with devtools::build_readme()
+# - Include a lint command (from https://github.com/r-lib/actions/tree/v2/examples#lint-workflow)
```

Comment:
We might still want linting to run automatically.  It occasionally catches bugs, and if we're going to check it before merging anyway, then it also reduces wait time.  Guess it might block merging if we want to ignore it, but I think we have the ability to override that anyway.